apps/extension/src/background.ts

```diff
@@ -18,3 +18,44 @@ chrome.runtime.onInstalled.addListener((details) => {
     });
   }
 });
+chrome.runtime.onMessage.addListener((message, _sender, sendResponse) => {
+  if (message.type === 'SAVE_BOOKMARK') {
+    const { url, title } = message.payload;
```

Comment:
해당 함수에서 첫 if문 블록이 길어서
```ts
sendResponse({ success: false, message: 'Unknown message type' });
return false;
```
아래있는 해당 로직을 파악하기가 쉽지 않을 것 같아요! 그래서 초기에 반대의 경우를 먼저 return 해주면 어떨까요??

```suggestion
chrome.runtime.onMessage.addListener((message, _sender, sendResponse) => {
if (message.type !== 'SAVE_BOOKMARK') {
  sendResponse({ success: false, message: 'Unknown message type' });
  return false;
}
```